Release/2.0.0

dkotter opened the pull request:
### Description of the Change

Version bump to 2.0.0 in preparation for a release.

_Please discard the branch name, the version was renumbered after the branch was pushed._

Closes #423

### How to test the Change

Verify files manually via VS Code or the GitHub UI

dkotter committed:
Version bump to 1.9.0

dkotter committed:
Update CHANGELOG

dkotter committed:
Update CREDITS.md

dkotter committed:
Update README.md

dkotter commented:
@jeffpaul In prepping this release, there are quite a few things going into it, more than I was thinking. We could bump to a 2.0.0 release if you want though I'm fine either way.

peterwilsoncc reviewed (commented):
These changes look good to me, I checked the zip file to verify the contents are as expected.

I'll leave approval to @jeffpaul pending the decision about whether to call this 1.9.0 or 2.0.0.

peterwilsoncc committed:
1.9.0 is 2.0.0 now.

peterwilsoncc commented:
Per discussion with Jeff in 1:1, I've updated all instances of 1.9.0 in this branch to 2.0.0 in 8920834b5f989724dba14c9c7f86c8c3346c5826

jeffpaul committed:
Update CHANGELOG.md

jeffpaul committed:
Update README.md

pulls up the OpenAI features to top of listing, fixes section headings, tweaks other readme bits and bobs

jeffpaul reviewed (approved):
Great work, very excited to see folks benefit from all this work, to get feedback on how we can iterate on these features and what additional things are desired next... exciting times now and EXCITING TIMES AHEAD!
